CHANGELOG.md

```
@@ -371,7 +376,11 @@ You can find the changelog for older versions there [here](https://github.com/TH
 
 [#31]: https://github.com/THM-Health/PILOS/issues/31
 [#75]: https://github.com/THM-Health/PILOS/issues/75
+<<<<<<< HEAD
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)